### ISSUE TITLE
onOutsideClick return false doesn't work

### ISSUE THREAD

**Issue by eldyvoon** [airbnb/react-dates#485]
How can I prevent the calendar being close when user did not select either start or endDate? I've set numberOfMonths to 2, it's strange to allow user to enter just one side of date in date range picker.

**Comment by majapw** [comment]
Like, if they click outside of the dropdown, you don't want to close it when one of the inputs still needs to be selected?

**Comment by eldyvoon** [comment]
Correct, there's a function but when I return false it doesn't have any
effect, the dropdown still got closed.

On 5 May 2017 12:05 AM, "Maja Wichrowska" <notifications@github.com> wrote:

Like, if they click outside of the dropdown, you don't want to close it
when one of the inputs still needs to be selected?

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/airbnb/react-dates/issues/485#issuecomment-299231697>,
or mute the thread
